src/foam/nanos/crunch/ui/CapablePayloadROView.js

```diff
@@ -115,7 +116,11 @@ foam.CLASS({
 
         // OPTION TWO
         // view.start().tag(this.VerticalDetailView, { data: data }).end()
-        view.start().add(data).end()
+        view.start()
+          .tag(this.FlexSectionedDetailView, {
+            data: this.capableObj.capablePayloads[i].data
+          })
+        .end() 
```

Comment:
```suggestion
        view
          .tag(this.FlexSectionedDetailView, {
            data: this.capableObj.capablePayloads[i].data
          })
```
dont need the wrapping .start().end() here as tag() adds it's own div